shell/platform/embedder/embedder.cc

```diff
@@ -662,7 +665,7 @@ FlutterEngineResult FlutterEngineShutdown(FlutterEngine engine) {
 }
 
 FlutterEngineResult FlutterEngineSendWindowMetricsEvent(
-    FlutterEngine engine,
+    FLUTTER_EMBEDDING_API(FlutterEngine) engine,
```

Comment:
Maybe call this `FLUTTER_API_SYMBOL`. Dunno, just find that clearer.

Reply:
Done.